client/my-sites/site-settings/form-seo.jsx

```diff
@@ -532,31 +600,42 @@ export const SeoForm = React.createClass( {
 							{ hasError( 'yandex' ) && this.getVerificationError( showPasteError ) }
 						</FormFieldset>
 						<FormFieldset>
-							<FormLabel htmlFor="seo_sitemap">{ this.translate( 'XML Sitemap' ) }</FormLabel>
-							<ExternalLink className="seo-sitemap" icon={ true } href={ sitemapUrl } target="_blank">{ sitemapUrl }</ExternalLink>
+							<FormLabel htmlFor="seo_sitemap">{ translate( 'XML Sitemap' ) }</FormLabel>
+							<ExternalLink
+								className="seo-sitemap"
+								icon={ true }
+								href={ sitemapUrl }
+								target="_blank">
+								{ sitemapUrl }
+							</ExternalLink>
 							<FormSettingExplanation>
-								{ this.translate( 'Your site\'s sitemap is automatically sent to all major search engines for indexing.' ) }
+								{ translate( 'Your site\'s sitemap is automatically sent to all major search engines for indexing.' ) }
 							</FormSettingExplanation>
 						</FormFieldset>
 					</Card>
 				</form>
 				<WebPreview
 					showPreview={ showPreview }
-				    onClose={ this.hidePreview }
-				    previewUrl={ siteUrl }
-				    showDeviceSwitcher={ false }
-				    showExternal={ false }
-				    defaultViewportDevice="seo"
+					onClose={ this.hidePreview }
+					previewUrl={ siteUrl }
+					showDeviceSwitcher={ false }
+					showExternal={ false }
+					defaultViewportDevice="seo"
 				/>
 			</div>
 		);
 	}
 } );
 
-const mapStateToProps = state => ( {
-	selectedSite: getSelectedSite( state ),
-	storedTitleFormats: getSeoTitleFormatsForSite( getSelectedSite( state ) )
-} );
+const mapStateToProps = ( state, ownProps ) => {
+	const { site = getSelectedSite( state ) } = ownProps;
```

Comment:
not sure how I feel about this defaulting assignment here. it makes a conceptual jump by not being a literal value, instead being a function call which depends on state.

wouldn't prevent this merge based on it, but definitely there's some hesitation that I want to think about it. for example, does `getSelectedSite( state )` get called if `ownProps` has a `site` variable?